setup.sh

```diff
@@ -42,9 +42,11 @@ check() {
     # make sure Docker client is pointed to the same place as the Triton client
```

Comment:
Thoughts on moving `setup.sh` into `examples/triton`? It doesn't seem like a requirement for `examples/compose` or `examples/multi-dc`.

Reply:
The multi-dc example's setup script calls `setup.sh` once per datacenter so there is a dependency between the two. Nothing preventing us from referencing it as `../triton/setup.sh` but it would lead to some confusion.

If the multi-dc example were named triton-multi-dc, or lived in `examples/triton/multi-dc` it might make the script dependency more obvious. Thoughts?

Reply:
Since this isn't a code library you can copy the `check()` function into your own shell script. Your script is superseding that setup anyway so owning it might help extend things later, like automatic setup across multiple profiles instead of leaving that up to the end user (removing those extra `export` steps over each DC). The latter isn't something we need to be concerned with ATM though.

Reply:
`check` was copied into `./setup-multi-dc.sh` and renamed to `generate_env`